app/services/exercise_service.rb

```diff
@@ -0,0 +1,20 @@
+class ExerciseService
```

Comment:
Great job here - @SullyBirashk , You and I need to remember to get these keys on our local